components/src/core/Drawer/DrawerNavItem/DrawerNavItem.tsx

```diff
@@ -137,39 +137,42 @@ const InfoListItemRoot = styled(InfoListItem, {
     name: 'drawer-nav-item',
     slot: 'info-list-item-root',
     shouldForwardProp: (prop) => prop !== 'drawerOpen' && prop !== 'active',
-})<Pick<DrawerNavItemProps, 'depth' | 'hidePadding' | 'icon'> & { drawerOpen: boolean; active: boolean }>(
-    ({ depth, hidePadding, icon, drawerOpen, active, theme }) => ({
-        '& .MuiListItemButton-root': {
-            // Have to specify both of these. JSS doesn't like to automatically flip the rule when it's calculated from a function
-            paddingLeft: theme.direction === 'rtl' ? theme.spacing(2) : calcNestedPadding(theme, depth),
-            paddingRight: theme.direction === 'ltr' ? theme.spacing(2) : calcNestedPadding(theme, depth),
-        },
-        [`& .${drawerNavItemClasses.nestedTitle}`]: {
-            fontWeight: 400,
-        },
-        [`& .${drawerNavItemClasses.title}`]: {
-            fontWeight: 400,
-        },
-        [`& .${drawerNavItemClasses.titleActive}`]: {
-            fontWeight: 600,
-        },
-        '& .BluiInfoListItem-title': {
-            opacity: hidePadding && !icon && drawerOpen ? 1 : hidePadding && !icon ? 0 : 'inherit',
-            transition: hidePadding && !icon ? theme.transitions.create('opacity') : '',
-        },
-        '& .BluiInfoListItem-subtitle': {
-            opacity: hidePadding && !icon && drawerOpen ? 1 : hidePadding && !icon ? 0 : 'inherit',
-            transition: hidePadding && !icon ? theme.transitions.create('opacity') : '',
-            color: !active && theme.palette.mode === 'dark' ? theme.palette.text.secondary : undefined,
-        },
-        '& .BluiInfoListItem-info': {
-            color: !active && theme.palette.mode === 'dark' ? theme.palette.text.secondary : undefined,
-        },
-        [`&. ${drawerNavItemClasses.ripple}`]: {
-            backgroundColor: theme.palette.primary.main,
-        },
-    })
-);
+})<
```

Comment:
I wonder where these new indents came from?  Kind of weird when code blocks just change like this.

Reply:
Unsure. My local prettier config may be off, but it passed CI prettier checks. I'm wondering if having the comment in there vs not is giving us a different read on the first line and then the rest follow for indentation parity.

Reply:
fixed